src/NATS.Client.Core/NatsSubOpts.cs

```diff
@@ -1,3 +1,5 @@
+using System.Threading.Channels;
```

Comment:
I think this `using` came in because of the XmlDoc, if preferred I can just use full type name in xmldoc?